_adr/87/index.adoc

```diff
@@ -1,48 +1,451 @@
+A list of the metrics included in the remote-write configuration can be found in the https://github.com/bf2fc6cc711aee1a0c2a/observability-resources-mk/blob/main/resources/prometheus/remote-write.yaml[Observability resources] repository.
+
+KAS Fleet Manager acts as a proxy to Observatorium.
+It calls the Observatorium _/query_ and _/query_range_ endpoints to retrieve metrics we consider as 'customer facing'.
+A list of the metrics retrieved by KAS Fleet Manager can be found in the https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/pkg/client/observatorium/api.go[observatorium] client within the project. When new metrics are added to this list, they become available in the KAS Fleet Manager _/metrics/query_ and _/metrics/query_range_ endpoints. In order to make new metrics available to the _/metrics/federate_ endpoint, their metadata must be added to this https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/internal/kafka/constants/metrics.go[file] in KAS Fleet Manager.
+
+KAS Fleet Manager controls the labels exposed for each metric returned in all endpoints. 
+Metrics retrieved from Observatorium are also filtered so that end users will only receive metrics for their own Kafka instance.
+Apart from these, no additional aggregation, filtering or modification of the metric is done by KAS Fleet Manager.
 
 ## Proposal
-// What is the decision being proposed
+### Metric and Label Naming
+All metric and label names must adhere to the policy stated in link:../../_ap/16/index.adoc[AP-16: Naming Policy for Customer Facing Metrics].
+
+### Kafka Metric Categories
+The customer facing Kafka metrics can be divided into multiple categories.
+Categorizing these metrics into logical groups can help us formulate the metric name and document it.
+
+|===
+|Category | Description | Sub-component (to be specified in the metric name)
+
+|Instance
+|General Kafka metrics not specific to any other sub-components.
+|instance
+
+|Broker
+|Metrics associated to Kafka brokers
+|broker
+
+|Topic
+|Metrics associated to Kafka topics
+|topic (Even though the topic metrics already have a 'topic' label, including this improves the clarity of the metric name)
+|===
+
+Examples of sub-component usage in the metric name:
+
+* For *Instance* metrics, the metric prefix would be `kas_instance_`.
+* For *Broker* metrics, the metric prefix would be `kas_broker_`.
+* For *Topic* metrics, the metric prefix would be `kas_topic_`.
+
+### Converted Metrics
+At minimum, each metric should have a label that identifies the Kafka instance if possible.
+This would be useful if a user has multiple Kafka instances.
+If they want to integrate all of their Kafka metrics into one monitoring system, the label(s) can act as an identifier.
+For Kafka metrics, we can use `instance_name` and `instance_id` to identify the Kafka instance the metric belongs to.
+
+#### Cluster Metrics
```

Comment:
I wouldn't include the Converted Metrics as part of the ADR.  I would describe the naming conventions and back those up with a few good examples (examples from the managed kafka domain is fine).  I'd move the Converted Metrics section to the Epic Brief.  Others might disagree.

Reply:
I've no problem moving these to the epic brief. wdyt @tombentley @pb82?

Reply:
Epic brief has been created, which is linked to this issue in JIRA: https://issues.redhat.com/browse/MGDSTRM-7080. 

I've moved the converted metrics into that and removed it from the ADR.